charts/daemonset/templates/cluster_role_binding.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.clusterrole.deploy }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ include "app.fullname" . }}-binding
```

Comment:
Not sure if there is a maximum length for this value - in the helpers we are truncating it to ensure max length is not exceeded but if adding another 8 chars I think that could be worth validating?  Also given the type is `ClusterRoleBinding` does this add useful context outside of fullname?

Reply:
Dropped the -binding suffix here